laravel/app/Http/Middleware/CheckForMaintenanceMode.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode as Middleware;
+
+class CheckForMaintenanceMode extends Middleware
+{
+    /**
+     * The URIs that should be reachable while maintenance mode is enabled.
+     *
+     * @var array
+     */
+    protected $except = [
+        //
```

Comment:
There seems to be a lot of arrays with a comment line in them. Any reason for this?

Reply:
This file should actually be deleted (which I'll do now), but in general, once again, these are based on the laravel 8 template. They're used for excluding certain routes from the middleware.